specs/sb/api.md

```diff
@@ -573,9 +573,9 @@ existing system.
 </thead>
 <tbody>
 <tr>
-  <td>organization_guid*</td>
+  <td>context_guid*</td>
```

Comment:
Interesting. I see what you mean. I'll talk to the folks I know are using this.

FWIW, we wouldn't be able to remove the org and space fields from the spec until v3 of the broker API, but we could introduce a replacement field and have CF send it as you have suggested, then start having consumers of the org and space fields move to the new one.